tests/RouteTest.php

```diff
@@ -17,8 +17,7 @@ public function testImmutablePath()
         $route = new Route();
         $route->path('/foo');
         $this->expectException(
-            'Aura\Router\Exception\ImmutableProperty',
```

Comment:
Why did you removed the second parameter? Is those removed from phpunit also ?

Reply:
I guess this probably will not be an issue as we use yoast/phpunit-polyfills . If possible you can revert back that change.

Reply:
I revert back that change. it's ok, but...

I think there is an issue here
The method expectException in yoast/phpunit-polyfills accept just one argument
https://github.com/Yoast/PHPUnit-Polyfills/blob/1.x/src/Polyfills/ExpectException.php#L61

The seconds argument in RouteTest is simply ignored.

If the second argument is the error message then it would be 'Aura\Router\Route::$path is immutable once set'
(Route::name method for exemple)
called with  expectExceptionMessage( $message )